yaksh/documentation/moderator_docs/creating_course.rst

```diff
@@ -37,21 +40,81 @@ Features in Courses
+                    
+                    * Quiz will have the same duration as that of the original quiz.
+                    * Quiz won't start if the course is inactive or the quiz time has expired.
+                    * You will be notified about quiz prerequisites.(You can still attempt the quiz though)
+        * **Add Quizzes/Lessons for <module-name>**
+            Click to add/change/delete lessons or quizzes.
+        * **My Courses**
+            Clicking on this will show all the courses created by you.
+        * **Allotted courses**
+            Clicking on this will show all the courses allotted to you.
+        * **Add New Course**
+            Clicking on this will open course form to create new course.
+        * **Add/View Quizzes**
+            Click to view all the created quizzes and add new quizzes.
+        * **Add/View Lessons**
+            Click to view all the created lessons and add new lessons.
+        * **Add/View Modules**
+            Click to view all the created modules and add new modules.
+        * **Design Course**
+            Click to add/delete/change modules.
+        * **Add Teacher**
             Clicking on Add teacher can let you add teachers for the course. The teachers can edit and modify only the specific course that are allotted to them.
-        * Teachers added to the course
+        * **Clone Course**
+            This will create a clone of the course for the moderator.
+        * **Teachers added to the course**
             This shows all the teachers added to a particular course.
-        * Download CSV for the entire course
+        * **Download CSV for the entire course**
             This downloads the CSV file containing the performance of all students in every quiz for a given course.
-        * Edit Course
+        * **Edit Course**
             Clicking on the edit course button will let you edit the details of an existing course.
+        * **Deactivate/Activate Course**
+            Clicking on this will deactivate or activate the course.
+
+
+Design a Course
+---------------
+    
+    Clicking on **Design Course** will show the below page.
+
+    .. image:: ../images/design_course.jpg
+
+    **Available Modules** contains all the modules that are not added to a course.
+
+    To add a module to the course select the checkbox beside every module and click **Add to course** button.
+
+    **Choosen Modules** contains all the modules that are added to a course.
+
+    Following parameters can be changed while designing a course:
+
+        **Order** - Order in which modules are shown to a student.
+
+            To change a module's order change the value in the textbox under **Order** column and click **Change order**.
+
+        **Check Prerequisite** - Check if previous module is completed. Default value is **Yes**.
+            For e.g. A student has to first complete **Demo module** to attempt **Python Module** if the value is **Yes**.
```

Comment:
For e.g., Assuming a course contains modules Demo Module and Python module in the given order; a student has to first complete **Demo module** to attempt **Python Module** if the value is **Yes**.